### ISSUE TITLE
Error deploying contract

### ISSUE THREAD

**Issue by TejasSatish** [PatrickAlphaC/ethers-simple-storage-fcc#87]
I am having issues deploying my contract. There seems to be some problem with either my abi or bin files.
I have tried deleting and recompiling them, but it doesn't seem to help.

I am running my node file with this command 
```node
node deploy.js
```

The file executes and prints out this error
```error
Error: missing revert data (action="estimateGas", data=null, reason=null, transaction={ "data": "0x608060405234801561000f575f80fd5b506101a78061001d5f395ff3fe608060405234801561000f575f80fd5b5060043610610034575f3560e01c80634abfa1631461003857806372f4d89414610068575b5f80fd5b610052600480360381019061004d919061011e565b610084565b60405161005f9190610158565b60405180910390f35b610082600480360381019061007d919061011e565b6100a8565b005b5f8181548110610092575f80fd5b905f5260205f20015f91509050805f0154905081565b5f604051806020016040528083815250908060018154018082558091505060019003905f5260205f20015f909190919091505f820151815f0155505050565b5f80fd5b5f819050919050565b6100fd816100eb565b8114610107575f80fd5b50565b5f81359050610118816100f4565b92915050565b5f60208284031215610133576101326100e7565b5b5f6101408482850161010a565b91505092915050565b610152816100eb565b82525050565b5f60208201905061016b5f830184610149565b9291505056fea26469706673582212205ee6936225406b263bd0f306d26fb889d03c99aa694681a4c0bd30a839ee62b864736f6c63430008140033", "from": "0x0155E3dE035F242559C62F65fC8A8a2EA1a96272", "to": null }, invocation=null, revert=null, code=CALL_EXCEPTION, version=6.6.0)
    at makeError (D:\Tejas\Code\organ-donation\node_modules\ethers\lib.commonjs\utils\errors.js:125:21)
    at getBuiltinCallException (D:\Tejas\Code\organ-donation\node_modules\ethers\lib.commonjs\abi\abi-coder.js:104:37)
    at AbiCoder.getBuiltinCallException (D:\Tejas\Code\organ-donation\node_modules\ethers\lib.commonjs\abi\abi-coder.js:201:16)
    at JsonRpcProvider.getRpcError (D:\Tejas\Code\organ-donation\node_modules\ethers\lib.commonjs\providers\provider-jsonrpc.js:599:43) 
    at D:\Tejas\Code\organ-donation\node_modules\ethers\lib.commonjs\providers\provider-jsonrpc.js:267:45
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```
The contract is printed as a JSON object below:

```json
{
  code: 'CALL_EXCEPTION',
  action: 'estimateGas',
  data: null,
  reason: null,
  transaction: {
    to: null,
    data: '0x608060405234801561000f575f80fd5b506101a78061001d5f395ff3fe608060405234801561000f575f80fd5b5060043610610034575f3560e01c80634abfa1631461003857806372f4d89414610068575b5f80fd5b610052600480360381019061004d919061011e565b610084565b60405161005f9190610158565b60405180910390f35b610082600480360381019061007d919061011e565b6100a8565b005b5f8181548110610092575f80fd5b905f5260205f20015f91509050805f0154905081565b5f604051806020016040528083815250908060018154018082558091505060019003905f5260205f20015f909190919091505f820151815f0155505050565b5f80fd5b5f819050919050565b6100fd816100eb565b8114610107575f80fd5b50565b5f81359050610118816100f4565b92915050565b5f60208284031215610133576101326100e7565b5b5f6101408482850161010a565b91505092915050565b610152816100eb565b82525050565b5f60208201905061016b5f830184610149565b9291505056fea26469706673582212205ee6936225406b263bd0f306d26fb889d03c99aa694681a4c0bd30a839ee62b864736f6c63430008140033',
    from: '0x0155E3dE035F242559C62F65fC8A8a2EA1a96272'
  },
  invocation: null,
  revert: null,
  info: {
    error: {
      message: 'VM Exception while processing transaction: invalid opcode',
      stack: 'RuntimeError: VM Exception while processing transaction: invalid opcode\n' +
        '    at exactimate (C:\\Program Files\\WindowsApps\\GanacheUI_2.7.1.0_x64__rb4352f0jd4m2\\app\\resources\\static\\node\\node_modules\\ganache\\dist\\node\\1.js:2:182136)',
      code: -32000,
      name: 'RuntimeError',
      data: [Object]
    },
    payload: {
      method: 'eth_estimateGas',
      params: [Array],
      id: 4,
      jsonrpc: '2.0'
    }
  }
}
```

**Comment by CaeSea** [comment]
I am also getting this error.

**Comment by PatrickAlphaC** [comment]
Can you:
1. Make this a discusson on the full repo? https://github.com/smartcontractkit/full-blockchain-solidity-course-js/
2. Follow this section for formatting questions? https://www.youtube.com/watch?t=19846&v=gyMwXuJrbJQ&feature=youtu.be

**Comment by KarenChuang** [comment]
Try install [solc@0.8.7-fixed](mailto:solc@0.8.7-fixed) and use solidity 0.8.7 version
